### DISCUSSION TITLE
New ui

### DISCUSSION THREAD

**Discussion by moodejb123** [dani-garcia/vaultwarden#4566] [General]
Hi, when will the new Bitwarden web ui be released to Vaultwarden

**Comment by jobritz** [comment]
It is released, which version do you have installed?

**Reply by moodejb123** [reply]
I have Version 2024.1.2 installed

**Reply by jobritz** [reply]
I mean your docker Version, try to install the newest release (1.30.5)

**Reply by moodejb123** [reply]
I do have the latest version installed but I can't see the new UI

**Reply by jobritz** [reply]
then it might be in the testing version

**Reply by BlackDex** [reply]
It's currently in the testing tagged images indeed.

**Reply by moodejb123** [reply]
Is there an estimated time when the new UI will be released to the latest version?

**Reply by BlackDex** [reply]
No

**Reply by moodejb123** [reply]
Ok thanks
